package.json

```diff
@@ -20,8 +20,8 @@
     "flat": "^5.0.0",
     "html-entities": "^1.2.1",
     "http-proxy": "^1.18.0",
-    "jose": "^1.10.1",
-    "lodash": "^4.17.20",
```

Comment:
Depandabot build for `lodash` update previously failed?
 https://github.com/Regional-Australia-Bank/ADR-Gateway/pull/45